udata/templates/embed-dataset.html

```diff
@@ -1,81 +1,149 @@
         padding: 1px;
         margin: 0 5px 0 0;
         background: #FFF;
     }
-    .dataset-card .dataset-description {
+    .dataset-card .description {
         margin: 0 5px;
         padding: 5px;
     }
+    .dataset-card .description a {
+        text-decoration: none;
+        color: #5E5E5E;
+        display: block;
+    }
+    .dataset-card .meta {
+        border-top: 1px solid #EEE;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        padding: 2px 5px;
+        color: #5E5E5E;
+    }
+    .dataset-card .meta a {
+        text-decoration: none;
+        color: #5E5E5E;
+    }
 </style>
 <div class="dataset-card">
-    {% if item.organization %}
-        <a class="dataset-organization"
-           href="{{ item.organization.external_url }}">
-            <img class="dataset-image"
-                 src="{{ item.organization.logo(70, external=True) }}"
-                 alt="{{ item.organization.name }}"
-                 height="70" width="70">
-        </a>
-    {% else %}
-        <a class="dataset-organization"
-           href="{{ item.external_url }}">
-            <img class="dataset-image"
-                 src="{{ ''|placeholder(external=True) }}"
-                 alt=""
-                 height="70" width="70">
-        </a>
-    {% endif %}
-    <h4 class="dataset-title">
-        <a class="dataset-link"
-           href="{{ item.external_url }}" title="{{ item.title }}">
-            {{ item.title }}
+    <h4 class="title">
+        <a href="{{ item.external_url or item.url }}" title="{{ item.title }}">
+            {{ item.title|mdstrip(80) }}
         </a>
     </h4>
-    <p class="dataset-description">
-        {{ item.description | mdstrip(250) }}
-    </p>
+    <div class="content">
+        <p class="source">
+        {% if item.territory %}
+            <a href="{{ item.territory.external_url }}">
+                <img src="{{ item.territory.logo_url(external=True)|placeholder(external=True) }}"
+                     alt="{{ item.territory.name }}"
+                     height="70" width="70">
+            </a>
+        {% elif item.organization %}
+            <a href="{{ item.organization.external_url }}">
+                <img src="{{ item.organization.logo(70, external=True) }}"
+                     alt="{{ item.organization.name }}"
+                     height="70" width="70">
+            </a>
+        {% else %}
+            <a href="{{ item.external_url }}">
+                <img src="{{ ''|placeholder(external=True) }}"
+                     alt=""
+                     height="70" width="70">
+            </a>
+        {% endif %}
+        </p>
+        <p class="description">
+            <a href="{{ item.external_url or item.url }}" title="{{ item.title }}">
+                {{ item.description | mdstrip(110) }}
+            </a>
+        </p>
+    </div>
+    <footer class="meta">
+        <p>
+            <a href="">🔃 {{ _('Embed') }}</a>
```

Comment:
Well, about that unicode thing. It was mostly a test and I kept it as a demo but I'm not sure we should keep those.

The issue is to have some pictos without requiring to download those from our server. My query budget to render an embed is as low as possible so no font awesome or similar. Maybe base64 embedded icons?